File: programacao/Ex31.c
#include<stdio.h>
int main()
{
    int a, b,c, auxiliar, i, n;
    a = 1;
    b = 1;
    c = 1;	
	getch ();
    printf("\nA serie de Bergamaschi ate 20 e:\n\n");
    printf("%d\n%d\n%d\n", a,b,c);
    for(i = 0; i < 17; i++) 
    {
        auxiliar = a + b + c;
        a = b;
        b = c;
        c = auxiliar;
        printf("%d\n", auxiliar);
    }
	return (0);
}

File: python/Ex31.py
a=1
b=1
c=1
print ("A serie de Bergamaschi ate 20 é:")
print ("1")
print ("1")
print ("1")
for i in range (0, 17, 1):
    auxiliar= a+b+c
    a=b
    b=c
    c=auxiliar
    print (auxiliar)